lib/clAdapters/office365-js.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+require('es6-promise').polyfill();
```

Comment:
`index.js` already calls the `polyfill()` method so don't need to do again here (https://github.com/CrossLead/crosslead-adapters/blob/82328b1c8a652de5d6f8246a857ec852d301f7a2/lib/index.js#L12)